examples/default03.vcl

```diff
@@ -11,6 +11,8 @@ backend httpbin_org {
   .ssl_sni_hostname = "httpbin.org";
   .ssl_cert_hostname = "httpbin.org";
   .ssl_check_cert = always;
+    .min_tls_version = "1.2";
```

Comment:
nits: please align the indent 😄

Reply:
🤦

Reply:
done in 65681b93800293e159eab1e516c2a3b8875ae26c